rds-sns-event-notification/README.md

```diff
@@ -0,0 +1,63 @@
+# AWS Service 1 to AWS Service 2
+
+RDS Event Subscriptions allow users to configure notifications for RDS Events (provided through an SNS topic). This template configures an event subscription for failure, low storage, and availability event categories for RDS Instances.
+
+Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>
```

Comment:
add the url to the pattern as instructions in creating the pattern

Reply:
Done